src/System.Runtime/tests/Helpers.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Reflection;
+using System.Reflection.Emit;
+
+namespace System.Tests
+{
+    public static class Helpers
+    {
+        public static Type NonRuntimeType()
```

Comment:
This is a fairly expensive operation.  I know it's just used in tests, but could you cache it in a static field?